package.json

```diff
@@ -9,7 +9,7 @@
     "build": "tsc",
     "lint": "standardx \"**/*.ts\" | snazzy",
     "test": "npm run lint && npm run build && npm run test-only",
-    "test-only": "cross-env TS_NODE_PROJECT=test/tsconfig.json tap --ts --no-browser --coverage-report=html --coverage-report=text --100 test/*.ts",
+    "test-only": "cross-env TS_NODE_PROJECT=test/tsconfig.json tap --ts --no-browser --coverage-report=html --coverage-report=text --100 --jobs=2 test/*.ts",
```

Comment:
This is limiting concurrency, because every single one of our tests spawns a pool of Workers and running them with full parallelism makes me hit memory limits locally :smile:

Reply:
I would recommend adding in a `.taprc` file. We start to have a bit too many options :P

Reply:
Done :+1: